elementary/monitor/data_monitoring/data_monitoring_report.py

```diff
@@ -238,21 +243,18 @@ def _get_lineage(self, exclude_elementary_models: bool = False) -> LineageSchema
         return self.lineage_api.get_lineage(exclude_elementary_models)
 
     def _get_test_results_and_totals(
-        self,
-        days_back: Optional[int] = None,
-        disable_passed_test_metrics: bool = False,
+        self, test_results_db_rows: List[TestResultDBRowSchema]
     ):
         try:
             tests_results, invocation = self.tests_api.get_test_results(
                 filter=self.filter,
-                days_back=days_back,
-                disable_passed_test_metrics=disable_passed_test_metrics,
+                test_results_db_rows=test_results_db_rows,
                 disable_samples=self.disable_samples,
             )
-            tests_info = []
+            tests_metadata = []
```

Comment:
I think it needs to be called `test_metadatas`.
The current name makes it sound like there's a single metadata for many tests.